python/taichi/lang/__init__.py

```diff
@@ -247,18 +247,30 @@ def static_print(*args, __p=print, **kwargs):
     __p(*args, **kwargs)
 
 
-def benchmark(func, repeat=100, args=()):
+def benchmark(func, repeat=300, args=()):
     import taichi as ti
     import time
-    for i in range(repeat // 3):
-        func(*args)  # compile the kernel first
+    func(*args)  # compile the kernel first
     ti.sync()
     t = time.time()
     for n in range(repeat):
         func(*args)
     ti.get_runtime().sync()
     elapsed = time.time() - t
-    return elapsed / repeat
+    avg = elapsed / repeat * 1000  # miliseconds
```

Comment:
We can add a similar comment at `4000` in the other file, or just unify all units to seconds.

Reply:
My quick guess on the 4 vs 4000 above is the unit issue. It makes sense to add comments there to suggest the units. 

Nit: could you please use uniform units all over these changed tests? Ideally, the whole framework should be independent of units and we can set our desired units before the simulation starts, but that sounds like a lot of work involved.

Reply:
I think miliseconds is a good unit since it's easier to display than seconds, or what about nanoseconds if we want to display in int?

Reply:
OK let me make myself clearer. Ultimately we should use either SI units or keep dimensionless over all the computation and only deal with the unit conversion in the input and output phase. That gives the freedom to choose units to the users. A dedicated module just for unit handling would be nice for that :)